examples/azure-tf-example/README.md

```diff
@@ -0,0 +1,253 @@
+? How would you like to set credential "subscription_id" environment variable
+? Enter the environment variable that will be used to set credential "subscription_id" AZURE_SUBSCRIPTION_ID
+? How would you like to set credential "tenant_id" environment variable
+? Enter the environment variable that will be used to set credential "tenant_id" AZURE_TENANT_ID
+Saving credential to /Users/jeremyrickard/.porter/credentials/porter-terraform-example.yaml
+```
+
+## Installing the Bundle
+
+Once you have built the bundle and generated a credential, you're ready to install the bundle! To do that, you'll use the `porter install` command:
+
+```bash
+$ porter install -c porter-terraform-example
+installing porter-terraform-example...
+executing porter install configuration from /cnab/app/porter.yaml
+Create an Azure Storage Account
+Starting deployment operations...
+Finished deployment operations...
+Emit the key in base64 encoded form
+Here is a the storage account key (base64 encoded) ==> %%A KEY%%
+Create Azure CosmosDB and Event Hubs
+Initializing Terraform...
+/usr/bin/terraform terraform init -backend=true -backend-config=access_key=******* -backend-config=container_name=portertf -backend-config=key=porter-terraform-example.tfstate -backend-config=storage_account_name=porterstorage -reconfigure
+
+Initializing the backend...
+
+Successfully configured the backend "azurerm"! Terraform will automatically
+use this backend unless the backend configuration changes.
+
+Initializing provider plugins...
+- Checking for available provider plugins on https://releases.hashicorp.com...
+- Downloading plugin for provider "azurerm" (1.31.0)...
+
+< OUTPUT TRUNCATED>
+```
+
+Installing the bundle will take some amount of time. As it is running you should see an output line like this:
+
+```
+Here is a the storage account key (base64 encoded) ==> <SOME STRING>
+```
+
+This is the account key that you'll need to later uninstall the bundle. It will be base64 encoded, so you'll need to decode it before using it to uninstall the bundle:
+
+```bash
+$ echo <SOMESTRING> | base64 -D
+```
+
+You can also obtain the value from the Azure portal.
+
+## Uninstalling the Bundle
+
+When you're ready to uninstall the bundle, simply run the `porter uninstall` command and provide the storage account key:
+
+```bash
+$ porter uninstall -c porter-terraform-example --param tf_storage_account_key=%%YOUR KEY VALUE%%
+uninstalling porter-terraform-example...
+executing porter uninstall configuration from /cnab/app/porter.yaml
+Remove Azure CosmosDB and Event Hubs
+Initializing Terraform...
+/usr/bin/terraform terraform init -backend=true -backend-config=access_key=W0ZdWXzsjRRZSTm4AnoKaYMmD9HjNsKpwqVp+14+mkNSaKJvp8jfUnTPuiWZIXVvs+P7dLtz2v227MYi0Ta4cA== -backend-config=container_name=portertf -backend-config=key=porter-terraform-example.tfstate -backend-config=storage_account_name=porterstorage -reconfigure
```

Comment:
Assuming this key is no longer valid 😅

Reply:
it is not :) I'll update this to remove it though.